themes/mixin-icons-fontawesome6/mixin.config.php

```diff
@@ -0,0 +1,227 @@
+<?php
+return [
+    'icons' => [
+        'defaultSet' => 'FontAwesome6',
+        'sets' => [
+            /**
+             * Define icon sets here.
+             *
+             * All sets need:
+             * - 'template': which template the icon renders with
+             * - 'src': the location of the relevant resource (font, css, images)
+             * - 'prefix': prefix to place before each icon name for convenience
+             *             (ie. fa fa- for FontAwesome, default "")
+             */
+            'FontAwesome6' => [
+                'template' => 'font',
+                'src' => 'vendor/fontawesome-free-6.6.0-web/css/all.min.css',
```

Comment:
Should we avoid the hard-coded version number so that it's easier to upgrade (e.g. if we can do the npm installation thing, copy the contents to a generic fontawesome-free-6 folder) or is it important to include the full number for cache-busting, etc.?